BZ1913075:  Improve the wording of exposed image registry route in UI (1.4.2) (#30171)
openshift/openshift-docs: modules/migration-adding-cluster-to-cam.adoc

.Prerequisites

> Optional: If you are using Azure snapshots to copy data:

Reviewer: It's not really optional. It's required if you use Azure, so you already had the restriction noted correctly. I'd revert this change.

.Prerequisites

Optional: If you are using Azure snapshots to copy data:

* You must provide the Azure resource group name when you add the source cluster.
* The source and target clusters must be in the same Azure resource group and in the same location.

.Procedure

. Log in to the cluster.
. Obtain a service account token:
+
[source,terminal]
----

> $ oc sa get-token <service_account> -n <namespace> <1>

Reviewer 2: @apinnick I am curious why you don't use "migration-controller"  service account here?

Author: @XinRedhat 
I wasn't sure whether this was one of those cases where the user could choose any service account with cluster-admin privileges. Does the user have to use migration-controller SA?

Reviewer 2: so far yes. you can see the doc https://docs.openshift.com/container-platform/4.6/migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.html#migration-adding-cluster-to-cam_migrating-4-2-4

Author: Thanks. I'll revert here.

. Click *Add cluster*.
. Fill in the following fields:

> * *Cluster name*: The cluster name may contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.

Reviewer: ```suggestion
* *Cluster name*: The cluster name can contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.
```

> ! may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.

* *URL*: Specify the API server URL, for example, `\https://<www.example.com>:8443`.
* *Service account token*: Paste the `migration-controller` service account token.

> * *Exposed route host to image registry*: Optional: If you are using direct image migration, specify the exposed route to the image registry of the source cluster, for example, `www.example.apps.cluster.com`.

Reviewer: ```suggestion
* *Exposed route host to image registry*: If you are using direct image migration, specify the exposed route to the image registry of the source cluster, for example, `www.example.apps.cluster.com`.
```

If there's another optional use case for this parameter, please explain it, too.

+
You can specify a port. The default port is `5000`.

> * *Azure cluster*: Optional: Select this option if you are using Azure snapshots to copy your data.

Reviewer: ```suggestion
* *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
```

* *Azure resource group*: This field appears if *Azure cluster* is selected. Specify the Azure resource group.
* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.

> * *CA bundle file*: This field appears if *Require SSL verification* is selected. Optional: If you created a custom CA certificate bundle file for self-signed certificates, click *Browse*, select the CA bundle file, and upload it.

Reviewer: Is this optional if you require SSL verification? If not:
```suggestion
* *CA bundle file*: This field is displayed if *Require SSL verification* is selected. If you created a custom CA certificate bundle file for self-signed certificates, click *Browse*, select the CA bundle file, and upload it.
```
